### DISCUSSION TITLE
Custom Read Date.

### DISCUSSION THREAD

**Discussion by XyberDAWG** [advplyr/audiobookshelf#2804] [Ideas]
I like to mark books I have read so I can easily find books I have t read in my library. But right now if I mark books I have read it adds them to my current listening history and skews my data. I'd like to be able to add custom read dates, or if I mark it read, there is an option to mark it without affecting this years data. 

If we could sync from our Goodreads account it would make it much quicker.

**Comment by nichwall** [comment]
You can do this by making books as read and editing the database to be whatever date you want it to be. Otherwise, related to https://github.com/advplyr/audiobookshelf/issues/1653.

A Goodreads integration is not planned due to them shutting down the public API.

**Comment by JBaby9783** [comment]
This is related. https://github.com/advplyr/audiobookshelf/issues/2630. Give it a thumbs up if you'd like.
